Update README.md

SunnyZhou-1024 opened the pull request:
Save user's time from figuring out what jsonnet is.

SunnyZhou-1024 committed:
Update README.md

Folyd commented on README.md:
```diff
@@ -35,6 +35,11 @@ you want to search, the extension will response the related search results insta
+# Installing jsonnet first if you havn't installed it yet, we employ jsonnet to generate manifest.json
+sudo apt update
+sudo apt install snapd
+sudo snap install jsonnet
```
I prefer a platform-agnostic `jsonnet` installation link here rather than a Linux install instruction. 

For example, we can say:

`jsonnet` is required before getting started. To install `jsonnet`, please check `jsonnet`'s [README](https://github.com/google/jsonnet#packages).

SunnyZhou-1024 committed:
Update README.md

I followed the instructions in jsonnet's README once, but it fail me, but it failed me. May be it just didn't work on my PC, I think it is worth telling users some special cases.
